web-services/map-reduce-embedded/src/main/java/datawave/security/system/EmbeddedCallerPrincipalProducer.java

```diff
@@ -7,7 +7,11 @@
 import javax.interceptor.Interceptor;
 
 import datawave.security.authorization.DatawavePrincipal;
-import org.infinispan.commons.util.Base64;
+import org.apache.commons.codec.binary.Base64;
```

Comment:
Does this allow us to remove the infinispan dependency from the POM?

Reply:
No - it looks like infinispan is used elsewhere.

Reply:
Looks like we can probably delete it from the map-reduce-embedded pom